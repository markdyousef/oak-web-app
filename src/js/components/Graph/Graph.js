import React, { Component, PropTypes } from 'react';

import css from './Graph.css';

class Graph extends Component {
    static propTypes = {
        data: PropTypes.shape({
            x: PropTypes.arrayOf(PropTypes.object).isRequired,
            y: PropTypes.arrayOf(PropTypes.object).isRequired
        })
    }
    constructor() {
        super();
        this.state = {};
    }
    render() {
        return (
            <div className={css.container}>
                Graph
            </div>
        );
    }
}

export default Graph;
