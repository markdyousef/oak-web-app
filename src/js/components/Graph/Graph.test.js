// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import Graph from './Graph';

test('renders Graph', () => {
    const props = {
        data: {
            x: [],
            y: []
        },
        type: 'line'
    };
    const component = renderer.create(
        <Graph {...props} />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
