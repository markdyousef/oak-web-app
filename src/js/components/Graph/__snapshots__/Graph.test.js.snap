exports[`test renders Graph 1`] = `
<div
  className={undefined} />
`;
